contracts/v6/OraclePrice.sol

```diff
@@ -62,28 +61,27 @@ contract OraclePrice is Ownable, KeeperCompatibleInterface {
         lastTimeStamp = block.timestamp;
         counter = counter + 1;
 
-        // We don't use the performData in this example
-        // performData is generated by the Keeper's call to your `checkUpkeep` function
-        performData;
         updateTargetPrice();
     }
     
     function updateTargetPrice() internal {
-        uint256 length = aggregatorContracts.length;
-        uint256[] memory oraclePrice;
+        uint256 length = oracleInfo.length;
         for(uint256 i=0; i<length; i++) {
-            int256 latestPrice = EACAggregatorProxy(aggregatorContracts[i]).latestAnswer();
-            uint8 decimals = EACAggregatorProxy(aggregatorContracts[i]).decimals();
-            uint256 restDec = SafeMath.sub(18, uint256(decimals));
-            latestPrice = int256(SafeMath.mul(uint256(latestPrice), 10**restDec));
-            oraclePrice[i] = uint256(latestPrice);
+            OracleInfo storage oracles = oracleInfo[i];
+            if(oracles.status == true) {
```

Comment:
What does status == true mean here?

Reply:
Currently, we have multiple tokens to get price like EUR, JPY etc and when you put status of any oracle false then we exclude that oracle and will not use that oracle.

Means we can exclude any oracle by status updates.

Reply:
rename status flag to ignoreChanges or something more descriptive.

Reply:
rename status flag to ignoreChanges or something more descriptive.

Reply:
Renamed to isActive